README.adoc

```diff
@@ -1,5 +1,8 @@
 # notifications-gw project
 
+To run the application or the testsuite you'll need Kafka running on port 9092.
```

Comment:
testsuite doesn't need to have kafka running. The tests already take care of starting one.

Reply:
True